parm/config/config.vrfy

```diff
@@ -104,6 +111,10 @@ fi
 
 export ens_tracker_ver=v1.1.15.5
 export HOMEens_tracker=$BASE_GIT/TC_tracker/${ens_tracker_ver}
+if [[ ${machine} = "JET" ]]; then
+   export ens_tracker_ver=v1.1.15.4
+   export HOMEens_tracker=/lfs1/NESDIS/nesdis-rdo2/David.Huber/TC_Tracker/TC_tracker.v1.1.15.4
+fi
```

Comment:
Let me know when you're ready for me to copy this to the glopara space or the TC_tracker repo has Jet support to build out-of-the-box (even better). Do you need any assistance from me on this external component? Should be able to remove most, if not all of, this block before this PR goes in (or a follow-up one if needed).

Reply:
Should I just go ahead and update the tracker version to 1.1.15.5 and use the tracker currently installed in glopara/git/TC_tracker/v1.1.15.5? It looks like that version has Jet support built into it.

Reply:
Yes, try out the v1.1.15.5 install on Jet. I forgot I installed that there last fall lol! Let Jiayi know if you encounter any problems. Thanks!